sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -937,6 +959,7 @@ void ProgramManager::addImages(pi_device_binaries DeviceBinary) {
     if (KSId == 0)
       KSId = getNextKernelSetId();
 
+    handleXilinxKernels(Img.get());
```

Comment:
The name is a little bit misleading since it will actually handle other kernels too (by doing nothing...)